commands/api/commands/http/endpoint.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package http
+
+import (
+	"context"
+
+	"github.com/go-kit/kit/endpoint"
+	"github.com/mainflux/mainflux/commands"
+)
+
+func pingEndpoint(svc commands.Service) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		req := request.(pingReq)
+
+		if err := req.validate(); err != nil {
+			return nil, err
+		}
+
+		greeting, err := svc.Ping(req.Secret)
+		if err != nil {
+			return nil, err
+		}
+
+		res := pingRes{
+			Greeting: greeting,
+		}
+		return res, nil
+	}
+}
+
+func getEndpoint(svc commands.Service) endpoint.Endpoint {
```

Comment:
Is it `get` or `view`? What convention did we use in other services? Please align.